_includes/integrations/react-tech-ref.md

```diff
@@ -0,0 +1,331 @@
+## TinyMCE React technical reference
+
+### Configuring the editor
+
+The editor accepts the following properties:
+
+```html
+<Editor
+  apiKey: 'your-api-key';
+  cloudChannel: '5-stable';
+  disabled: false;
+  id: 'uuid';
+  init: {% raw %}{{{% endraw %}  {% raw %}}}{% endraw %};
+  initialValue: '';
+  inline: false;
+  onEditorChange: '';
+  plugins: '';
+  tagName: 'div';
+  textareaName: '';
+  toolbar: '';
+  value: '';
+/>
+```
+
+None of the configuration properties are **required** for `tinymce-react` to work. Specify a Tiny Cloud API key using `apiKey` to remove the `This domain is not registered...` warning message.
+
+#### `apiKey`
+Tiny API key. Required for deployments using the Tiny Cloud to provide the TinyMCE editor.
+
+To register for a Tiny Cloud API key, visit the [sign-up page](https://www.tiny.cloud/signup/).
+
+**Default value:** `no-api-key`
+
+**Type:** String
+
+##### Example: `apiKey`
+
+```html
+<Editor
+  apiKey: 'your-api-key';
+/>
+```
+
+#### `cloudChannel`
+
+**Default value:** `5-stable`
+
+**Possible values:**  `5-stable`, `5-testing`, `5-dev`
+
+Changes the TinyMCE build used for the editor to one of the following cloud channels:
+
+- `5-stable` (**Default**): The current enterprise release of TinyMCE.
+- `5-testing`: The current release candidate for the next enterprise release of TinyMCE.
+- `5-dev`: The nightly-build version of TinyMCE.
+
+Such as:
+
+```js
+<Editor apiKey='your-api-key' cloudChannel='5-dev' init={% raw %}{{{% endraw %} /* your other settings */ {% raw %}}}{% endraw %} />
+```
+For information TinyMCE development channels, see: [Specifying the TinyMCE editor version deployed from Cloud - dev, testing, and stable releases]({{site.baseurl}}/cloud-deployment-guide/editor-plugin-version/#devtestingandstablereleases).
+
+#### `disabled`
+The `disabled` property can dynamically switch the editor between a "disabled" (read-only) mode (`true`) and the standard editable mode (`false`).
+
+**Default value:** `false`
+
+**Possible values:**  `true`, `false`
+
+##### Example: `disabled`
+
+```html
+<Editor
+  disabled: true;
+/>
```

Comment:
```
<Editor
  disabled={true}
/>
```
This is the case for some other places as well
`prop='string'` with strings and
`prop={value}` in all other cases

Reply:
Just got a crash-course on react syntax from @glennpenridge . Updating to correct syntax now.